src/find-me-announcements/entities/find-me-announcement.entity.ts

```diff
@@ -0,0 +1,75 @@
+import { Column, CreateDateColumn, Entity, JoinTable, ManyToMany, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+
```

Comment:
pusta linia

Reply:
Plugin do eslinta, który układa importy sortuje też importy spoza projektu (node_modules) a te z projketu i dzieli jest jedną pustą linią - wydaje mi sie to organizacyjnie poprawne 🍡  

<img width="831" alt="image" src="https://user-images.githubusercontent.com/39082174/162438614-2da544b8-a5e1-4eb3-97fc-acfbd4d02e84.png">